### PR TITLE
fix(spotifyControls): add album/cover null checks (local files)

### PR DESCRIPTION
**Pull request by D3SOX**
Fixes local files

image

after 

image

### CODE REVIEW

**Comment by D3SOX** [comment]
I just noticed this breaks when the user previously opened the cover view, need to push another fix

File: src/plugins/spotifyControls/PlayerComponent.tsx [hunk 1]
```diff
@@ -233,18 +233,22 @@ function Info({ track }: { track: Track; }) {
...
-            onClick={() => setCoverExpanded(!coverExpanded)}
-            onContextMenu={e => {
-                ContextMenu.open(e, () => <AlbumContextMenu track={track} />);
-            }}
-        />
+        <>
+            {img && (
+                <img
+                    id={cl("album-image")}
+                    src={img?.url}
```
Comment (Vendicated) [inline src/plugins/spotifyControls/PlayerComponent.tsx:240]:
```suggestion
                    src={img.url}
```

File: src/plugins/spotifyControls/PlayerComponent.tsx [hunk 2]
```diff
@@ -233,18 +233,22 @@ function Info({ track }: { track: Track; }) {
...
+                    onContextMenu={e => {
+                        ContextMenu.open(e, () => <AlbumContextMenu track={track} />);
+                    }}
+                />
+            )}
+        </>
     );
 
-    if (coverExpanded) return (
+    if (coverExpanded && img) return (
```
Comment (Vendicated) [inline src/plugins/spotifyControls/PlayerComponent.tsx:251]: uh how could u expand cover if img is null?

Comment (D3SOX) [reply src/plugins/spotifyControls/PlayerComponent.tsx:251]: Expand cover, then play a local file

**Pull request merged**